app/repositories/story_repository.rb

```diff
@@ -1,7 +1,10 @@
+require_relative "../helpers/url_helpers"
 require_relative "../models/story"
 require_relative "../utils/sample_story"
 
 class StoryRepository
+  extend UrlHelpers
```

Comment:
Switching to `include UrlHelpers` adds `expand_absolute_urls` and `normalize_url` as instance methods (instead of class methods) and causes `NoMethodError: undefined method 'expand_absolute_urls' for StoryRepository:Class`. Unfortunately, that doesn't appear to be it. :worried: